config/prow/plugins.yaml

```diff
@@ -222,7 +222,6 @@ lgtm:
   - kubernetes-sigs/aws-ebs-csi-driver
   - kubernetes-sigs/bom
   - kubernetes-sigs/boskos
```

Comment:
This entry here overwrote the config for store_tree_hash below. So now we have a separate array element to set both store_tree_hash and review_acts_as_lgtm